[wasm-runtime] Error: Cannot find module './dist/fs-proxy.cjs'

After upgrading to `@napi-rs/wasm-runtime` v1.0.2, we start to get the `Cannot find module './dist/fs-proxy.cjs'` error.

```bash
node -e "require('@napi-rs/wasm-runtime')"
```

It seems like there is no `fs-proxy.cjs` in `dist/`.

<img width="410" height="297" alt="Image" src="https://github.com/user-attachments/assets/4dc73cb6-d213-4b87-b9c0-bdac80130d65" />

Not sure if this is related with https://github.com/napi-rs/napi-rs/pull/2810.

--- a/wasm-runtime/package.json
+++ b/wasm-runtime/package.json
@@ -21,8 +21,8 @@
     "runtime.cjs",
     "runtime.js",
     "fs-proxy.js",
-    "dist/*.js",
-    "dist/*.cjs"
+    "dist/*.cjs",
+    "dist/*.js"
   ],
   "devDependencies": {
     "@rollup/plugin-alias": "^5.1.1",

--- a/wasm-runtime/package.json
+++ b/wasm-runtime/package.json
@@ -21,7 +21,8 @@
     "runtime.cjs",
     "runtime.js",
     "fs-proxy.js",
-    "dist/*.js"
+    "dist/*.js",
+    "dist/*.cjs"
   ],
   "devDependencies": {
     "@rollup/plugin-alias": "^5.1.1",

